contentcuration/contentcuration/frontend/shared/feedbackApiUtils.js

```diff
@@ -0,0 +1,148 @@
+// Helper functions and Utils for creating an API request to
+// Feedback mechanism endpoints
+import { v4 as uuidv4 } from 'uuid';
+import client from './client';
+
+export const FeedbackTypeOptions = {
+  imported: 'imported',
+  rejected: 'rejected',
+  previewd: 'previewd',
+  showmore: 'showmore',
+  ignored: 'ignored',
+  flagged: 'flagged',
+};
+
+// This is mock currently, fixed value of URL still to be decided
+export const FLAG_FEEDBACK_EVENT_URL = '/api/flagged/';
```

Comment:
Django has a way of providing access to the URLs in the frontend, where you can reference them with by name, without hard coding the actual path/URL like this. There are some existing examples in the codebase, particularly `resources.js` I believe.